gaj23/face-it
# Loading Message Component

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

Adds a LoadingMessage component to render with an animated image and text when the fetch request is gathering data. 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.

The rendering code in the App component is refactored, along with added state properties to make the loading screen compatible.  There was a previous bug where the default state was on the error message, so I refactored the state to add an 'isFetching' and 'error' property, so the default was on the loading component unless an error occurred during the fetch request.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App.js, LoadingImg.js, Loading.js, LoadingImg.css, Loading.css

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Start the backend server in the 'face-it-server' repo and run `node server.js`

Open the project directory and run `npm start` in the terminal to view the page in your browser on http://localhost:3000.  To see the loading page in action, you will need to go into your dev tools and slow down the speed of your connection enough to see the page load on refresh.  


#### Checklist:
